docs/articles/integrations-webhooks.md

```diff
@@ -1,99 +1,104 @@
 # Integrations Webhooks
 
-The Testkube Dashboard allows you to easily create and manage the Webhooks defined for your Environment.
+Webhooks are request-driven and external – exposing HTTP endpoints that run Testkube actions when called by external systems, such as CI/CD tools, GitHub, Slack, or other services. You might
+create a Webhook, for example, to notify your team if a critical API test fails so the team can fix the issue immediately. If this test finishes with a *Failed* status,
+Testkube (based on your Webhook configuration) sends an HTTP POST request to a Slack Webhook URL which then posts a message in the *Alerts* channel. The Slack Webhook URL is a private entrance (of sorts) to the *Alerts* channel,
+automatically turning data into a chat message for your team. This allows Testkube to communicate with Slack without the need for you to manually copy/paste this message. 
 
-:::tip
-Check out the [Webhooks](/articles/webhooks) document to learn how Webhooks can be used in Testkube.
-:::
-
-Webhooks are created under an Environment and are managed from the "Webhooks" tab in the integrations panel:
+You can create and manage **Webhooks** in your **Environment** via the **Testkube Dashboard**, selecting **Integrations** from the left navigation pane then proceeding to the **Webhooks** tab. 
+For additional information, read [Webhooks](/articles/webhooks).
 
-![Webhooks](../img/integrations-webhooks.png)
+![Webhooks](../articles/images/testkube-dashboard-integrations-webhooks.png)
 
 ## Creating a New Webhook
 
-Select the **Create a new webhook** button to create a new Webhook. 
-
-![Create New Webhook](../img/create-new-webhook.png)
+You can select **Add a new Webhook** to access the **Create a Webhook** modal.
 
 ### Webhook Condition
 
-You will initially be prompted for:
+The **Create a Webhook** modal displays:
+
+* **Name** – Unique identifier that must represent a valid Kubernetes name, as Webhooks are stored as Custom Resource Definitions (CRDs) in your cluster.
+* **Resource Identifier** – The *who* watched, such as a specific test. Resources follow the official Kubernetes selector pattern. Multiple selectors are grouped by the **AND** operator.
+* **Triggered events** – The *what* or specific Kubernetes action that causes the Testkube Webhook to run, such as a *Failed* test status.
 
-- the **Name** of the Webhook - must be a valid Kubernetes name as Webhooks are stored as CRDs in your cluster.
-- the **Resources Identifier** for which resources to monitor, can either be a list of Kubernetes Labels or a specific resource name.
-- the **Triggered events** that will execute the Webhook.
+**EXAMPLE**
 
-![Create New Webhook Modal 1](../img/create-webhook-modal-1.png)
+When a Testkube test finishes with a *Failed* status, Testkube sends an HTTP POST request to a Slack Webhook URL which then posts a message in your *Alerts* Slack channel to notify team members.
 
```

Comment:
we have a complete example of how to set this up at https://docs.testkube.io/articles/slack-integration - perhaps link there?